demos/motmetrics4norfair/motmetrics4norfair.py

```diff
@@ -0,0 +1,125 @@
+import sys
+import os.path
+import numpy as np
+import norfair
+from norfair import Detection, Tracker, Video, lib_metrics, video
+
+# get arguments from terminal
+(
+    videos,
+    input_path,
+    save_path,
+    make_video,
+    show_metrics,
+    save_pred,
+) = lib_metrics.get_arguments()
+
+frame_skip_period = 1
+detection_threshold = 0.01
+distance_threshold = 0.4
+
+Accumulator = lib_metrics.accumulators(input_path=input_path)
+
+# proccess every video
+for testing_video in videos:
+    input_path_det = os.path.join(input_path, testing_video, "det/det.txt")
```

Comment:
`input_path_det` is not really an input in this context, its just a path to the detection file, so it should be named accordingly. Something like ` detections_file_path` should work I think.